theme/nuxt.config.js

```diff
@@ -177,14 +172,41 @@ export default {
     },
   },
   i18n: {
-    defaultLocale: process.env.LOCALE_DEFAULT || 'en_US',
```

Comment:
i18n configs were updated to match new nuxt-i18n-localization plugin that was introduced in @vue-storefront/nuxt 2.3.4